pkg/session_recording/recordings.jsx

```diff
@@ -238,16 +243,51 @@
                             navigateToItem={this.navigateToRecording.bind(this, r)}/>);
             }
             return (
+                <div>
                 <Listing.Listing title={_("Sessions")}
                                  columnTitles={columnTitles}
                                  emptyCaption={_("No recorded sessions")}
-                                 fullWidth={false}>
+                                 fullWidth={false}
+                                 >
                     {rows}
                 </Listing.Listing>
+                </div>
             );
         }
     };
 
+    var Datepicker = class extends React.Component {
+        constructor(props) {
+            super(props);
+            this.handleDateChange = this.handleDateChange.bind(this);
+        }
+
+        componentDidMount() {
+            var funcDate = this.handleDateChange;
+            let elemId = '#' + this.props.element;
+            $(elemId).datepicker({
```

Comment:
I think the proper way to do this is React "refs": https://facebook.github.io/react/docs/refs-and-the-dom.html